mongodb/src/test/java/org/apache/calcite/test/MongoAdapterIT.java

```diff
@@ -703,6 +700,7 @@ public Void apply(ResultSet resultSet) {
   /** MongoDB's predicates are handed (they can only accept literals on the
    * right-hand size) so it's worth testing that we handle them right both
    * ways around. */
+  @Ignore("broken; [CALCITE-2351] is logged to fix it")
```

Comment:
I've already removed this line. I marked it as ignore because it fails on my PC on current master, but @vvysotskyi mentioned that it works fine on his PC.

Reply:
@vvysotskyi Just wondering if you tried this with the changes in this PR just to make sure it's not the cause of the failure.

Reply:
@michaelmior, I have run MongoAdapterIT tests on master only when tried to reproduce CALCITE-2351. I'll run MongoAdapterIT tests before merging this.

Reply:
@vvysotskyi If you're able to run the tests and then merge once you're satisfied, that would be great. Thanks!